android/java/v2/AmpliApp/app/src/main/java/com/example/ampliapp/SegmentDestinationPlugin.java

```diff
@@ -14,10 +14,10 @@
 public class SegmentDestinationPlugin extends DestinationPlugin {
     android.content.Context context;
     Analytics analytics;
-    String SEGMENT_API_KEY;
-    public SegmentDestinationPlugin(android.content.Context appContext, String segmentAPIKey) {
+    String SEGMENT_WRITE_KEY;
```

Comment:
nit: instance variable I think would be better as camelCase, and leading "segment" isn't needed since it's limited to this classes scope.
```suggestion
    String writeKey;
```